src/test/java/Task30/BaseTest.java

```diff
@@ -5,19 +5,19 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.openqa.selenium.WebDriver;
 import page.LoginPage;
-import page.MailPage;
-
+import screenShot.ScreenShot;
 
 public class BaseTest {
 
     public static final String SITE_NAME = "https://mail.yandex.com/";
     protected LoginPage loginPage;
 
     @BeforeEach
-    public void init() {
+    public void init(){
         WebDriver driver = WebDriverSingleton.getDriver();
         driver.manage().window().maximize();
         driver.get(SITE_NAME);
+        ScreenShot.makeScreenShot(driver, "C://ScreenShots//HomePage.png");
```

Comment:
please, do not use absolute path, use path from content root and save it somewhere in project, ex.: target/screenshots. And it would be nice to differentiate the file name so that the method can be reused in different parts of the project, getting different files, and not writing everything to one file, erasing the previous ones